content/articles/2021-08-01-dbee-nvim.md

```diff
@@ -0,0 +1,72 @@
+# My first vim plugin!
+
+Have you ever had the necessity of check a query while you are writing a python
+wrapper? If so, this post is for you.
+
+Many times I had the necessity of checking a query while I'm writing a function,
+debugging or testing it. That's why I wrote my first vim (well, actually a
```

Comment:
Many times I have had the necessity of checking a query while I'm writing,
debugging or testing a function.